owncloudDomain/src/main/java/com/owncloud/android/domain/user/model/UserQuotaStatus.kt

```diff
@@ -0,0 +1,20 @@
+package com.owncloud.android.domain.user.model
+
+enum class UserQuotaStatus(val value: String) {
```

Comment:
You called it everywhere `state`, not `status`. Let's keep consistency across the code